ocs_ci/utility/utils.py

```diff
@@ -2363,3 +2363,33 @@ def set_aws_region(region=None):
     log.debug("Exporting environment variable AWS_REGION")
     region = region or config.ENV_DATA['region']
     os.environ['AWS_REGION'] = region
+
+
+def modify_csv(csv, replace_from, replace_to):
+    """
+    Modify the CSV
+
+    Args:
+        csv (str): The CSV name
+        replace_from (str): The pattern to replace from in the CSV
+        replace_to (str): The pattern to replace to in the CSV
+
+    """
+    data = (
+        f"oc -n openshift-storage get csv {csv} -o yaml | sed"
+        f" 's,{replace_from},{replace_to},g' | oc replace -f -"
+    )
+    log.info(
+        f"CSV {csv} will be modified: {replace_from} will be replaced "
+        f"with {replace_to}.\nThe command that will be used for that is:\n{data}"
+    )
+
+    temp_file = NamedTemporaryFile(
+        mode='w+', prefix='csv_modification', suffix='.sh'
+    )
+
+    with open(temp_file.name, 'w') as t_file:
+        t_file.writelines(data)
+
+    run_cmd(f"chmod 777 {temp_file.name}")
+    run_cmd(f"sh {temp_file.name}")
```

Comment:
why not run_oc_cmd ?

Reply:
for some reason, while running this command with `run_cmd`, it doesn't work, I tried every possible way but only this way it works. 
Probably it has to do with `sed`